src/features/import/hooks/useColumn.ts

```diff
@@ -0,0 +1,11 @@
+import { Column } from '../utils/types';
+import { updateColumn } from '../store';
+import { useAppDispatch } from 'core/hooks';
+
+export default function useColumn() {
```

Comment:
I think this hook should either be renamed, or modified to return something different, or even both. 😊 

1. Either rename it to `useUpdateColumn()`, or
2. Have it return an object with a named `updateColumn` function on it, or
3. Do number 2, but also rename it to `useColumnMutations()`

I think I prefer number 2.